dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Bind.java

```diff
@@ -64,21 +71,51 @@ public class Bind extends CamelCommand {
                         required = true)
     String sink;
 
+    @CommandLine.Option(names = { "-p", "--prop", "--property" },
```

Comment:
We only use a single and double slash for the commands in camel-jbang

Reply:
I was copying this from Run command: https://github.com/apache/camel/blob/main/dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Run.java#L235

Reply:
Ah yeah we should have that changed now after the 4.4 LTS so it can be consistent. You are welcome to fix that as well. Maybe in another PR and add a note in the upgrade guide 4.x

Reply:
so to be sure it is about removing the `-p` option, right? I can do that for the new options in this PR already, np

Reply:
yeah

Reply:
done